kevm-pyk/src/tests/conftest.py

```diff
@@ -31,8 +29,8 @@ def pytest_addoption(parser: Parser) -> None:
         help='Run only this specific specification (skip others)',
     )
     parser.addoption(
-        '--kompiled-targets-dir',
-        type=dir_path,
+        '--kompiled-target-dir',
```

Comment:
May I ask you to not change this name, as our performance script [uses](https://github.com/runtimeverification/haskell-backend/blob/6e5778899e91ff3c951221bdb620c69b5ec41aa3/scripts/performance-tests-kevm.sh#L105) the old one?

Reply:
Done.